EMS/core-bundle/src/Controller/ChannelController.php

```diff
@@ -32,7 +33,9 @@ public function index(Request $request): Response
     {
         $table = $this->dataTableFactory->create(ChannelDataTableType::class);
 
-        $form = $this->createForm(TableType::class, $table);
+        $form = $this->createForm(TableType::class, $table, [
+            'reorder_label' => t('type.reorder', ['type' => 'channel'],'emsco-core')
```

Comment:
Then the pattern 'channel' is not translatable ?

Reply:
channel is just the type for the translation key: 

```yaml
type:
    reorder: |-
        {type, select,
            channel {Reorder channels}
            other   {Reorder}
        }
```

More info about this format https://symfony.com/doc/current/reference/formats/message_format.html#selecting-different-messages-based-on-a-condition